charts/synthetics-private-location/values.yaml

```diff
@@ -104,3 +104,7 @@ hostAliases: []
 #  - ip: "10.0.0.1"
 #    hostnames:
 #    - "host.domain.com"
+
+# enableStatusProbes -- Enable both liveness and readiness probes
+enableStatusProbes: false
```

Comment:
Does this parameter depend on minimal private location image version to work? if so, let's mention in in the doc string + in the changelog.